README.md

```diff
@@ -78,9 +76,9 @@ $ mv buildx ~/.docker/cli-plugins/docker-buildx
 
 ## Building with buildx
 
-Buildx is a Docker CLI plugin that extends the `docker build` command with the full support of the features provided by [Moby BuildKit](https://github.com/moby/buildkit) builder toolkit. It provides the same user experience as `docker build` with many new features like creating scoped builder instances and building against multiple nodes concurrently.
+Buildx is a Docker CLI plugin that extends the `docker build` command with the full support of the features provided by [Moby BuildKit](https://github.com/moby/buildkit) builder toolkit. It provides the same user experience as `docker build` with many new features like creating scoped builder instances and building against multiple nodes concurrently
 
-After installation, buildx can be accessed through the `docker buildx` command.  `docker buildx build` is the command for starting a new build.
+After installation, buildx can be accessed through the `docker buildx` command with Docker 19.03.  `docker buildx build` is the command for starting a new build. With Docker 19.03-, you still need to run `~/.docker/cli-plugins/docker-buildx` to replace `docker buildx` in the following description.
```

Comment:
"With Docker versions older than 19.03 buildx binary can be called directly to access the `docker buildx` subcommands."